sdk/analytics/src/main/java/com/klaviyo/analytics/networking/KlaviyoApiClient.kt

```diff
@@ -152,13 +152,13 @@ internal object KlaviyoApiClient : ApiClient {
             } catch (exception: JSONException) {
                 wasMutated = true
                 Registry.log.wtf("Invalid persistent queue JSON", exception)
-                Registry.log.debug(it)
+                Registry.log.info(it)
                 emptyArray<String>()
             }
         }?.forEach { uuid ->
             Registry.dataStore.fetch(uuid).let { json ->
                 if (json == null) {
-                    Registry.log.error("Missing request JSON for $uuid")
+                    Registry.log.debug("Missing request JSON for $uuid")
```

Comment:
hmm why go from error to debug?

Reply:
Previously, all these levels were only here for us to see, because in a published build there was no logging. 
I didn't think this was worth logging in the client's application, so I lowered it so it won't be visible by the default logging level in a release build.